spec/jobs/upload_users_job_spec.rb

```diff
@@ -0,0 +1,34 @@
+describe UploadUsersJob do
+  let(:course) { create :course }
+
+  context 'when running as a background job' do
+    it 'runs successfully as a background job' do
+      file = 'spec/fixtures/files/students/students.csv'
+      let!(:job_args) { [Student, course, File.read(file), nil] }
+      include_examples 'background job'
+    end
+  end
+  context 'when run with a correctly formatted file' do
+    before :each do
+      @file = 'spec/fixtures/files/students/students.csv'
```

Comment:
Please use `let` variables instead of instance variables in specs whenever possible